src/utils/prettyNum.ts

```diff
@@ -0,0 +1,58 @@
+// Chrome doesn't yet support icelandic for Number.toLocaleString (2.9.2020)
+const i18n = {
+	is: [',', '.'],
+	en: ['.', ','],
+} as const;
+type SupportedLanguages = keyof typeof i18n;
+
+export type PrettyNumOptions = {
+	/** Max number of decimals to render */
+	decimals?: number;
+	/** Should decimals be uniformly applied to all numbers */
+	fixedDecimals?: boolean;
+	/** Default: `true` */
+	leadingZero?: boolean;
+	/** Default: `'en'`
+	 *
+	 * If your language isn't supported use the `splitters` option instead.
+	 */
+	lang?: SupportedLanguages;
+	/** The tokens to use for a) decimals and b) thousands
+	 *
+	 * Examples Icelandic uses `[',', '.']` and English uses `['.', ',']`
+	 *
+	 * Default: `['.', '']`
+	 */
+	splitters?: [string, string] | [string];
+	// Supported by TS 4.0 https://devblogs.microsoft.com/typescript/announcing-typescript-4-0-rc/#labeled-tuple-elements
+	// splitters?: [decimal: string, thousands: string] | [decimal: string];
+};
+
+export const prettyNum = (number?: string | number, opts: PrettyNumOptions = {}) => {
+	const {
+		decimals = 0, // is this the right default?
+		fixedDecimals,
+		leadingZero = true,
+		lang = 'en',
+		// NOTE: explicit `splitters` value always trumps the `lang` option
+		splitters = i18n[lang] || [],
+	} = opts;
+	const numFloat = parseFloat('' + number);
+	if (isNaN(numFloat)) {
+		console.error('Invalid numer: ', number);
+		return 'error';
```

Comment:
```suggestion
		if (typeof process != undefined && process.env.NODE_ENV !== 'production') {
			console.error("prettyNum can't format:", number);
		}
		return '';
```

Ef við ætlum ekki að springa á invalid input - þá skulum við bara skila snyrtilegum tómum streng ''.
Það er leiðinlegt að prenta sýnilegt `"NaN"` eða `"error"` út í viðmótið.

Ég er pínu á báðum áttum hvort svona fall eigi að vera að logga villur.
En ef við gerum það held ég við ættum að láta þann kóða hverfa í production.